nautobot/project-static/.node-version

```diff
@@ -0,0 +1 @@
+22
```

Comment:
This mimics what's in the package.json file:

https://github.com/nautobot/nautobot/blob/792817263c83fbdc47149e6ce3333959ac2d5f72/nautobot/project-static/package.json#L57

https://github.com/shadowspawn/node-version-usage?tab=readme-ov-file#compatibility-testing